packages/common/src/networks/data.json

```
@@ -45,6 +45,29 @@
     "multicall2Address": "0x5BA1e12693Dc8F9c48aAD8770482f4739bEeD696",
     "multicall3Address": "0xcA11bde05977b3631167028862bE2a173976CA11"
   },
+  {
+    "id": "bnb",
+    "chainId": 56,
+    "name": "Bnb Chain",
```

```suggestion
    "name": "BNB Chain",
```
As it's the full name string rather than the variable or key name.